lake/oneshot_cls.py

```diff
@@ -263,7 +263,12 @@ def main():
     recall_target = torch.from_numpy(np.array(recall_target)).to(device)
 
     # Reset to saved model
-    model.load_state_dict(torch.load(pretrained_model_path))
+    model_dict = model.state_dict()
+    pretrained_dict = torch.load(pretrained_model_path)
+    pretrained_dict = {k: v for k, v in pretrained_dict.items() if k in model_dict}
+    model_dict.update(pretrained_dict)
```

Comment:
this line looks redundant?

Reply:
Any thoughts on this one?

Reply:
I am going to assume you are commented about `model_dict.update(pretrained_dict)` - if so, then yes it is redundant! 

I was confused by the GitHub showing the same code snippet but with different comments